src/loaders/genome_collection/parse_tool_results.py

```diff
@@ -163,6 +185,9 @@ def _update_docs_with_meta_info(res_dict, meta_lookup, check_genome):
 
         res_dict[genome_id].update({names.FLD_UPA_MAP: upa_dict})
 
+        # add Genome WS object metadata info
+        res_dict[genome_id].update(_get_genome_obj_meta(meta_info.get(loader_common_names.GENOME_OBJ_INFO_KEY, [{}])))
```

Comment:
Hm, this makes me a little nervous - if we neglected to redownload old data or forgot to download the genomes this would silently pass

Reply:
Plus _get_genome_obj_meta will fail even without my change since the [-1] expects a non zero length array

Reply:
👍 raise error when we got empty genome meta.

Reply:
I guess this is ok, although in the future it's possible we might have assembly only collections, but I suppose we can deal with that then